app/models/SaveSubmittedProcessResponse.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Json, OFormat}
+
+// TODO confirm what is returned as id is just the ocelot process id
+case class SaveSubmittedProcessResponse(id: String)
```

Comment:
Good to include Approval in the naming as we're likely to have other submitted processes

Reply:
The external-guidance submitted end point only returns the id - it has no concept of approval at the moment - not sure if it should.  Maybe we need  design call to discuss how we see the different services working, as currently the external-guidance is pretty dumb and just takes what it receives and persists

Reply:
We haven't really tied this down yet. Do we use a single collection for both processes pending approval and process that have been approved? As far as I am aware approved is not the same as published. If we have both pending and approved processes in the collection perhaps something like in-progress or under-development would be more suitable.

Reply:
Yes, PublishedProcesses will have a number of states throughout the flow and what is returned from the call to external-guidance will become clearer when we know the actual flow and what the UI will be submitting/expecting.  I'll leave as is for now.